manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TheStellaTeam.Stella
+PackageVersion: 6.7.1
+Installers:
+- InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-x64.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: 03C7A64C9616D38BB21891B7F9C976734C2994C9A127AFD75FFF396F6037476A
+- InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-win32.exe
+  Architecture: x86
+  InstallerType: inno

```

```suggestion
  InstallerType: inno
  AppsAndFeaturesEntries:
  - DisplayName: Stella 6.7.1
```
AppsAndFeaturesEntries are required for matching since no version is written to control panel